Assets/Scripts/trial/FieldTrial.cs

```diff
@@ -19,13 +20,26 @@ public FieldTrial(InputField[] fields) : base(-1, -1)
         {
             _fields = fields;
 
-            // This block of code basically calls the directory picker to select the configuration file.
-            while (true)
+            // This block of code will use the default configuration file if
+            // using webGL or Android (can add others in if statement) 
+            // Otherwise calls the directory picker to select the configuration file.
+
+            string[] files = Directory.GetFiles(Application.streamingAssetsPath + "/Default_Config/");
```

Comment:
Use var instead of string[]. Also, consider adding "/Default_Config/" as a value under the constants class.

Reply:
Changed to var, I was wondering what the advantage of changing to constants class is since it's only being used here in the codebase and no where else. Is it a best practice?